### PR TITLE
Add ability to view Forex Rates

### PR DESCRIPTION
**Pull request by KamuelaFranco**

### CODE REVIEW

File: src/components/ForexRatesTab.js [hunk 1]
```diff
@@ -39,22 +43,86 @@ const styles = (theme) => ({
...
-      <p>Forex Rates Tab</p>
-    </div>
+    <Grid className={classes.root} container spacing={0}>
+      <Grid item xs={12}>
+        <ForexRateEntry />
+      </Grid>
+      <Grid item xs={12}>
+        <ForexRatesTable forexRates={forexRates} />
+      </Grid>
+      {console.log(snackBarParams)}
```
Comment (msk-) [inline src/components/ForexRatesTab.js]: Did you intend for this to remain?

Comment (msk-) [reply src/components/ForexRatesTab.js]: If so, consider adding a little more context to what you're printing. So it's a little more obvious to a reader, and so it's easier to find in the source (yes, I know there are sourcemaps etc., but that does slow things down a bit, and IIRC the suggestion here will still display nicely in the console).
```suggestion
      {console.log('Error in forex rates component. Message parameters:', snackBarParams)}
```

File: package.json [hunk 2]
```diff
@@ -41,11 +41,11 @@
     "@storybook/preset-create-react-app": "^1.5.2",
     "@storybook/react": "^5.3.9",
     "@testing-library/react": "^9.4.0",
-    "eslint": "^6.8.0",
+    "eslint": "^6.1.0",
```
[the remark this answers is not in the document]
Comment (KamuelaFranco) [reply package.json]: Using the automatic peer dependencies tool for Airbnb ESLint produced that combination of versions known compatible.

File: src/components/ForexRatesTab.js (hunk 1 above)
Comment (KamuelaFranco) [reply src/components/ForexRatesTab.js]: I wanted it to remain so that the linter wouldn’t get angry that I hadn’t used snackbarParams yet.

**Pull request merged**